docker/Dockerfile.build_env

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:20.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ARG USERNAME=devuser
+
+RUN apt-get update \
+ && apt-get upgrade -y \
+ && apt-get install -y \
+                wget git python2 libc6-i386 \
+                make npm
+
+# build files expect "python", but it gets installed as "python2".
+# create symbolic link to fix this.
+RUN ln -s /usr/bin/python2 /usr/bin/python
```

Comment:
Unless you explicitly need `python2` to compile, the symlink is not required. If you read Ubuntu 20.04 release notes, they say no package is going to provide `/usr/bin/python` anymore, if you install `python3` you get the `/usr/bin/python3` binary, the same thing for `python2`.

If you need the symlink you should install `python-is-python3` package or `python-is-python2`, available on the official repositories. But as mentioned above, I believe `python3` should work as well, unless there's an explicit requirement for `python2`.

Reply:
I was having some troubles using `python3` (if you look at the SDK code, you'll note they tried to wedge in `python3` support on top of original `python2` code), but I cannot replicate that at this moment, so I will switch it to `python3`. But, you do need a symlink to the python executable [because of this line](https://github.com/openshwprojects/OpenBK7231T/blob/master/platforms/bk7231t/bk7231t_os/build.sh#L82). I figured it is easier and less impactful to create the symlink than to change the original SDK code. As far as using `python-is-python3`, if you look at what it does, the `.deb` is just a symlink too. With said, I will change it to `python-is-python3` because it's the same thing.